Seagate/cortx-utils
# Cortx-32105: Codacy pep257

# Problem Statement
- PEP 257 issues

# Design

(D104): Missing docstring in public package.
(D105): Missing docstring in magic method.
(D107): Missing docstring in init.
(D203): 1 blank line required before class docstring (found 0).
(D211): No blank lines allowed before class docstring (found 1).
(D212): Multi-line docstring summary should start at the first line.
(D213): Multi-line docstring summary should start at the second line.

D211 and D203 are conflicting, All D203 should be ignored, and all D211 are wrongly-reported.
D212 and D213 are conflicting. All D212 should be ignored and all D213 are wrongly-reported.
Also following error codes will be ignored:
(D203), (D212), (D107), (D104), (D105).

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing
- Are test cases updated along with code changes due to Enhancements/Bugs [Y/N]:N/A
- Confirm that new test cases are added to regression and sanity plan files and relevant feature plan files [Y/N]: N/A
- Confirm that Test Cases are added for new features added [Y/N]: N/A
- Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: N/A
- Confirm Testing was performed with installed RPM/K8s deployment [Y/N]:  N/A

# Review Checklist 
####  Before posting the PR please ensure:
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]: N/A
- [ ] New package/s added to setup.py?:  N/A
- [x] All the tests performed should be mentioned before Resolving a JIRA. :  N/A

# KVstore/Confstore feature (changes/fixes) checklist
#### Confirm changes are made for:  N/A

# Documentation
- [ ] Changes done to WIKI / Confluence page:  N/A
